lib/changelog/arc_ecto/schema.ex

```diff
@@ -0,0 +1,75 @@
+defmodule Changelog.Arc.Ecto.Schema do
```

Comment:
We should add a comment at the top of this file stating that this is taken almost verbatim from `arc_ecto/lib/schema.ex`, but with minor adjustments to support the `allows_url` option.